### ISSUE TITLE
WebCam Streaming

### ISSUE THREAD

**Issue by RisKatOr** [DanielnetoDotCom/YouPHPTube#601]
WebCam Stream is a great feature, status is Online but shows black screen, how can I use this feature?
![webcam](https://user-images.githubusercontent.com/4102352/38806693-c34fe338-4182-11e8-9093-330b28a5affa.jpg)

**Comment by DanielnetoDotCom** [comment]
I have disabled it a while ago, it was not working properly

**Comment by gamersalpha** [comment]
+1

**Comment by RisKatOr** [comment]
After activating SSL on Website, Webcam started to work. It shows the status as Online, Stats also working but does not send live stream.

![webcam](https://user-images.githubusercontent.com/4102352/38836989-f9251a22-41d8-11e8-995b-8950a842f71f.jpg)

![stats](https://user-images.githubusercontent.com/4102352/38837033-22027aac-41d9-11e8-8752-e8afbabbf6e7.jpg)

**Comment by DanielnetoDotCom** [comment]
That is why, I need a better flash RTMP publisher

This one is not transmiting correct

![image](https://user-images.githubusercontent.com/18009293/38841970-8978acea-41bd-11e8-942a-142ccd791604.png)

**Comment by RisKatOr** [comment]
Yeah, something is not going well. I can broadcast with webcam and I can watch the broadcast with a different player but my own player and m3u8 do not work yet.

![webcam](https://user-images.githubusercontent.com/4102352/38920046-010d2f72-42fb-11e8-99f7-bea7d90ca04c.jpg)

![stats](https://user-images.githubusercontent.com/4102352/38920054-06e48440-42fb-11e8-87f5-318804256a8b.jpg)

**Comment by DanielnetoDotCom** [comment]
Hi, it will now work on that page, to be able to make it work you will need to refresh the page, and if you refresh you will loose the webcam connection.
try to open another page and see if your webcam works on your YPT

**Comment by RisKatOr** [comment]
No, it does not work. :(

![live](https://user-images.githubusercontent.com/4102352/38930077-41acefec-4317-11e8-99ff-2c5b64d1ac25.jpg)

**Comment by DanielnetoDotCom** [comment]
Yeah, this webcam RTMP publish does not work fine.
that is why it was disabled

**Comment by RisKatOr** [comment]
Yes, but we will find a solution.

**Comment by DanielnetoDotCom** [comment]
I appreciate that

**Comment by RisKatOr** [comment]
I will open it again when/if I find a solution.

**Comment by hustler203** [comment]
Any update in webcam usage
